events/admin.py

```diff
@@ -40,6 +41,8 @@ class EventAdmin(admin.ModelAdmin):
         'turnover',
         'absolute_price_change',
         'price_change',
+        'created_date',
+        'created_by',
```

Comment:
in Główne I think, also: (modification)
- is_moderated/activated column- every user submission should be reviewed?
- mod_date
- mod_by
- ...or even list of that changes? (new table in db)